user_guide_src/source/concepts/factories.rst

```diff
@@ -9,32 +9,84 @@ Factories
 Introduction
 ============
 
-Like ``Services``, ``Factories`` are an extension of autoloading that helps keep your code
-concise yet optimal, without having to pass around object instances between classes. At its
+What is Factories?
```

Comment:
Looking at the sentences following this, `Factories` is treated as a plural word.
```suggestion
What are Factories?
```